doc/source/whatsnew/v0.24.0.txt

```diff
@@ -164,6 +164,40 @@ array, but rather an ``ExtensionArray``:
 This is the same behavior as ``Series.values`` for categorical data. See
 :ref:`whatsnew_0240.api_breaking.interval_values` for more.
 
+.. _whatsnew_0240.enhancements.join_with_two_multiindexes:
+
+Joining with two multi-indexes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As of Pandas 0.24.0 the :func:`Dataframe.join` can be used to join multi-indexed ``Dataframe`` instances on the overlaping index levels (:issue:`6360`)
```

Comment:
Mention that this applies to `pd.merge` as well?

Reply:
Typically we prefer just :func:\`DataFrame.join\` rather than "the :func:...". Also, since this is a whatsnew, you can drop "As of Pandas 0.24.0"